nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,118 @@
+const express = require("express");
+const app = express();
+// Parse URL-encoded bodies (as sent by HTML forms)
+app.use(express.urlencoded({ extended: true }));
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+//add
+app.get("/calculator/add", (req, res) => {
+  const firstValue = req.query.firstNumber;
+  const secValue = req.query.secondNumber;
+  console.log(typeof secValue);
```

Comment:
Generally try to remove console.log statements that are not useful anymore. They are helpful while developing, but can be confusing in production :)